src/base64-inl.h

```diff
@@ -182,6 +211,96 @@ inline size_t base64_encode(const char* src,
   return dlen;
 }
 
+
+#if (defined(__x86_64) || defined(__x86_64__)) && \
+    (defined(__GNUC__) || defined(__GNUC))
+#pragma GCC target("avx512vl", "avx512vbmi")
+#include <immintrin.h>
```

Comment:
Include this at the top, you're including it inside a C++ namespace here.

Reply:
Will modify it. Thanks!